cloudformation/jolly-roger.yaml

```diff
@@ -698,6 +698,9 @@ Resources:
                     timeout connect 5s
                     timeout client 50s
                     timeout server 50s
+                    timeout http-request 20s
```

Comment:
`haproxy` knows about websockets, right?  Just making sure this won't force us to fall back to XHR